pkgs/development/python-modules/pass2csv/default.nix

```
@@ -0,0 +1,22 @@
+{ buildPythonApplication, fetchFromGitHub, python-gnupg, lib}:
+
+buildPythonApplication rec {
+  pname = "pass2csv";
+  version = "0.3.1";
+
+  src = fetchFromGitHub {
+    repo = pname;
+    owner = "reinefjord";
+    rev = "26f796afbab1048e592b282fd2deceffee7d6e51";
+    sha256 = "sha256:0bp5chbbssanprncrx2dvgn7rvm6h1w6pclxpr5jfhvdm14cinsc";
+  };
+
```

```suggestion
  # Remove once 0.3.2 is released
  postPatch = ''
    cat > setup.py << EOF
    from setuptools import setup
    if __name__ == "__main__":
        setup()
    EOF
  '';

```
From https://github.com/reinefjord/pass2csv/commit/c21ce19ff6b4f6d058734495915a9873538fe733